docs/source/versioning.rst

```diff
@@ -73,45 +101,23 @@ specification.
   For example, the pre-release of version ``1.2.3`` would be
   ``1.2.3-dev+a2c4``.
 
-Relationship of version numbers
--------------------------------
-
-The major number in the **standard version** is incremented whenever
-the major number in the **file format version** is incremented.
```

Comment:
This breaks the link between the specification/core-schema major version and the file format version. See https://github.com/asdf-format/asdf-standard/wiki/The-revenge-of-2.0.0-Roadmap#remove-link-between-format-and-schemas